src/matchers.ts

```diff
@@ -519,17 +543,17 @@ export async function matchesLlmRubric(
     defaultProvider,
     'llm-rubric check',
   );
-  const resp = await finalProvider.callApi(prompt, {
-    prompt: {
-      raw: prompt,
-      label: 'llm-rubric',
-    },
-    vars: {
+  const resp = await callProviderWithContext(
```

Comment:
IMO this function is an unnecessary abstraction; if its goal is to spread context before the other arguments, I'd prefer something like `serializeCallApiContext(...)` instead.